src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSelector.js

```diff
@@ -199,8 +199,9 @@ function MoneyRequestParticipantsSelector({
     const headerMessage = OptionsListUtils.getHeaderMessage(
         newChatOptions.personalDetails.length + newChatOptions.recentReports.length !== 0,
         Boolean(newChatOptions.userToInvite),
-        searchTerm,
+        searchTerm.trim(),
         maxParticipantsReached,
+        _.some(participants, (participant) => participant.login.toLowerCase().includes(searchTerm.trim().toLowerCase())),
```

Comment:
Looks like this caused a regression the login field is only available on user accounts but not on workspaces, this lead to a crash https://github.com/Expensify/App/issues/27864

Reply:
We should also cover the workspace case
<img width="377" alt="Screenshot 2023-09-20 at 3 45 02 PM" src="https://github.com/Expensify/App/assets/16493223/0e2c0b0a-302d-442a-b009-895c571f4c67">